conf/app.ini

```diff
@@ -439,6 +439,11 @@ SCHEDULE = @every 24h
 ; Archives created more than OLDER_THAN ago are subject to deletion
 OLDER_THAN = 24h
 
+; Synchronize external user data
```

Comment:
What user information is synced ? Could this comment be expanded to tell more about the feature ?